app/package.json

```diff
@@ -19,15 +19,13 @@
     "cookies": "^0.8.0",
     "eth-hooks": "^4.2.21",
     "googleapis": "^100.0.0",
-    "history": "^5.3.0",
```

Comment:
can we definitely remove these dependencies? I'm getting an error on 'npm run dev' after running npm install because of lack of history